pkg/scheduler/priorities_test.go

```diff
@@ -356,7 +354,7 @@ func TestNewNodeLabelPriority(t *testing.T) {
 			label:    test.label,
 			presence: test.presence,
 		}
-		list, err := prioritizer.CalculateNodeLabelPriority(test.pod, FakePodLister(test.pods), FakeMinionLister(api.NodeList{Items: test.nodes}))
+		list, err := prioritizer.CalculateNodeLabelPriority(nil, nil, FakeMinionLister(api.NodeList{Items: test.nodes}))
```

Comment:
test.Pod was never populated and was always the default. The function under test didn't actually use it, so I passed in a nil pointer instead of constructing a pod just to point to it.

Same for test.Pods.